go.mod

```
@@ -3,88 +3,70 @@ module github.com/hashicorp/vault-csi-provider
 go 1.19
```

May want to bump this too?

```suggestion
go 1.20
```